src/components/pages/movie-page/use-movie-page.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchMovie, fetchMovieImages, fetchMovieCast, fetchMovieRecommend, toggleCastList } from '../../../redux/movieSlice';
+
+
+export const useMoviePage = (id) => {
+  const dispatch = useDispatch();
+  const lang = useSelector(state => state.appReducer.lang);
```

Comment:
lang could be placed in a separate hook and be reusable in other components

Reply:
Fixed!